src/components/partial-string-match/PartialStringMatch.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import {shallow} from 'enzyme';
+import * as escapeRegExp from 'escape-string-regexp';
+import * as React from 'react';
+import {escape} from 'underscore';
+import {PartialStringMatch} from './PartialStringMatch';
+
+describe('PartialStringMatch', () => {
+    const testString = 'test-string';
+
+    it('should render an empty span if the wholeString is a falsy value', () => {
+        [undefined, ''].forEach((falsyVal) => {
+            expect(shallow(<PartialStringMatch wholeString={falsyVal} />).find('span').prop('dangerouslySetInnerHTML').__html)
+                .toBe('');
+        });
+    });
+
+    it('should render the wholeString unchanged if partialMatch is undefined', () => {
+        expect(
+            shallow(<PartialStringMatch wholeString={testString} />)
+                .find('span')
+                .prop('dangerouslySetInnerHTML')
+                .__html,

```

Comment:
You can probably just check `.html()` instead of `.prop('dangerouslySetInnerHTML').__html`, no?

Reply:
good point, will do before merging

Reply:
finally the `html()` function includes the `<span></span>` wrapper which we don't want here. I'll leave it like that.